code/modules/clothing/suits/armor.dm

```diff
@@ -325,6 +325,13 @@
 	slowdown = 0.7
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 
+// Marauder Corps / SyndieParatrooper Suit
+/obj/item/clothing/suit/armor/swat/gorlex
+	name = "tactical armour suit"
+	desc = "A tactical full-body armour suit, dubbed the \"NOVA\". Spaceproof, with a decent armour and emblazed with black and red colours. Property of Gorlex Security Consulting, LLC."
+	icon_state = "marauder_suit"
+	armor = list(MELEE = 35, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 90, FIRE = 100, ACID = 100, WOUND = 10)
```

Comment:
I only just realized this but we now use armor datum rather than directly adding the armor values to the suits. Have a look at my other comment to see what that datum looks like, and on this same file to see how it is implemented.